basiclibrary/src/main/java/basiclibrary/Library.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basiclibrary;
+
+import java.util.Random;
+import java.util.Arrays;
+
+public class Library {
+//    public boolean someLibraryMethod() {
+//        return true;
+//    }
+    public static void main(String[] args) {
```

Comment:
we don't want to have a main method in a library file, but it look like you were just wanting some visual output. you can also do this by printing in your test file.